src/index.html

```diff
@@ -11,18 +11,87 @@
     <header class="container">
         <nav>
             <ul>
-                <li><img src="./assets/d20.png"></li> 
-                <li>Lorem</li>
-                <li>Ipsum</li>
-                <li>Dolor</li>
+                <li>Comments</li>
+                <li>Reservation</li>
+                <li>Likes</li>
             </ul>
         </nav>
     </header>
-    <h1>This is a test!</h1>
-    <button>Press Me!</button>
+    
+    <!-- Item 1
+    <div class="item">
+        <h3>Item 1</h3>
+        <p>Likes: <span class="likesCount" data-item="item1">0</span></p>
+        <button class="likeButton" data-item="item1"><i class="fa fa-thumbs-up"></i></button>
+    </div>
+    
+    <!-- Item 2 -->
+    <div class="item">
+        <h3>Item 2</h3>
+        <p>Likes: <span class="likesCount" data-item="item2">0</span></p>
+        <button class="likeButton" data-item="item2"><i class="fa fa-thumbs-up"></i></button>
+    </div>
```

Comment:
Remember we have to work on generating and displaying items dynamically, instead of directly into our HTML. Try generating a loop that allows you to print this result.